ingestion/src/metadata/examples/workflows/rest.yaml

```diff
@@ -3,9 +3,9 @@ source:
   serviceName: openapi_rest
   serviceConnection:
     config:
-      type: REST
+      type: Rest
```

Comment:
why are we changing type "Rest"?

Reply:
cc: @harshsoni2024

Reply:
Actually when we used service name in all caps, we get multiple errors in OM apis, airflow connections while doing ingestion from UI. for e.g. In the connection class [func](https://github.com/open-metadata/OpenMetadata/blob/f3c107025a722e09012e7912a264074f8a5acf4d/ingestion/src/metadata/ingestion/api/parser.py#L70), it parses service name in capitalized format. We had to apply patch on multiple parts for the rest service so we had followed same naming conventions as we did in all other dashboard, database connectors.